### DISCUSSION TITLE
Jpeg files

### DISCUSSION THREAD

**Discussion by mlolcu** [szimek/signature_pad#678] [Q&A]
Dear friends,
excuse me I do not know Java, I am a Visual FoxPro developer and would like to include Signature Pad into one application.
I have this code running rightly:

`    <script>
        $(document).ready(function () {
            var signaturePad = new SignaturePad(document.getElementById('signature-pad'));

            $('#wwWebPanel1_Ok').click(function () {
                var data = signaturePad.toDataURL('image/png');
                $('#txtFirma').val(data);
            });
        })
    </script>
  `
This saves Png files.
Now I need to save Jpeg files, I did read documentation and tried this code, but unfortunately it creates black images:

`
  <script>
        $(document).ready(function () {
            var signaturePad = new SignaturePad(document.getElementById('signature-pad'));
            canvas,{backgroundColor: 'rgb(255, 255, 255)'};

            $('#wwWebPanel1_Ok').click(function () {
                var data = signaturePad.toDataURL('image/jpeg');
                $('#txtFirma').val(data);
            });
        })
    </script>
`

Please how is the right code?
Thank you very much for support

**Comment by UziTech** [comment]
The options object is the second parameter of the constructor.

```js
signaturePad = new SignaturePad(
  document.getElementById('signature-pad'),
  {
    backgroundColor: 'rgb(255, 255, 255)'
  }
);
```

**Comment by mlolcu** [comment]
Many thanks, Tony!
You are very kind.
Sincerely

Luca Moioli


Il giorno mar 13 dic 2022 alle ore 07:12 Tony Brix ***@***.***>
ha scritto:
